core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -173,9 +173,22 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
       }
 
       CommandLineUtils.checkRequiredArgs(parser, options, consumerConfigOpt, producerConfigOpt)
-      if (List(whitelistOpt, blacklistOpt).count(options.has) != 1) {
-        println("Exactly one of whitelist or blacklist is required.")
-        System.exit(1)
+
+      val useNewConsumer = options.has(useNewConsumerOpt)
+      if (useNewConsumer) {
+        if (options.has(blacklistOpt)) {
+          println("blacklist can not be used when using new consumer in mirror maker. Use whitelist instead.")
```

Comment:
Doesn't look like we use println anywhere else here, just normal logging calls. We should do the same here.